osmosis/assetlist.json

```diff
@@ -1117,6 +1117,55 @@
         "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/persistence/images/stkosmo.png",
         "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/persistence/images/stkosmo.svg"
       }
+    },
+    {
+      "description": "The PAGE token is used for actions in the PageDAO NFT literary ecosystem and for DAO governance.",
+      "denom_units": [
+        {
+          "denom": "ibc/23A62409E4AD8133116C249B1FA38EED30E500A115D7B153109462CD82C1CD99",
+          "exponent": 8,
+          "aliases": [
+            "PAGE"
+          ]
+        },
+        {
+          "denom": "PAGE.grv",
+          "exponent": 8
+        }
+      ],
+      "type_asset": "ics20",
+      "base": "ibc/23A62409E4AD8133116C249B1FA38EED30E500A115D7B153109462CD82C1CD99",
+      "name": "PageDAO",
+      "display": "Page",
+      "symbol": "PAGE.grv",
+      "traces": [
+        {
+          "type": "ibc",
+          "counterparty": {
+            "chain_name": "gravity",
+            "base_denom": "page",
+            "channel_id": "channel-10"
+          },
+          "chain": {
+            "channel_id": "channel-144",
+            "path": "transfer/channel-144/gravity0x60e683C6514Edd5F758A55b6f393BeBBAfaA8d5e"
+          }
+        }
+      ],
+      "images": [
+        {
+          "image_sync": {
+            "chain_name": "gravitybridge",
+            "base_denom": "page"
```

Comment:
'page' is not the base denom on gravitybridge. Please replace with 'gravity0x60e683C6514Edd5F758A55b6f393BeBBAfaA8d5e'